app/src/main/java/com/raghav/spacedawn/ui/common/ItemLoadState.kt

```diff
@@ -0,0 +1,80 @@
+package com.raghav.spacedawn.ui.common
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Card
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.themeadapter.material.MdcTheme
+import com.raghav.spacedawn.R
+
+@Composable
+fun ItemLoadState(
+    message: String,
+    isProgressBarVisible: Boolean,
+    isCtaVisible: Boolean,
+    modifier: Modifier = Modifier,
+    ctaListener: () -> Unit = {}
+) {
+    Card(
+        modifier,
+        shape = RoundedCornerShape(20.dp)
```

Comment:
Nit - It's always good to follow the [4-point grid system](https://medium.com/@aratidube12lns/4-point-grid-system-for-more-consistent-interface-design-efea81dea3f3#:~:text=What%20is%20the%204%2Dpoint,padding%2C%20margins%20and%20line%20heights.) whenever possible to create a consistent-looking UI.

Reply:
Ok, will read about it and apply the same principle in future.